frontend/components/panels/VaultLiquidationPanel.stories.js

```diff
@@ -0,0 +1,86 @@
+import { storiesOf } from '@storybook/vue';
+import { action } from '@storybook/addon-actions';
+import faker from 'faker';
+import BigNumber from 'bignumber.js';
+import { getAllCollateralTypes } from 'auctions-core/src/constants/COLLATERALS.ts';
+import VaultLiquidationPanel from './VaultLiquidationPanel';
+import { generateFakeLiquidationLimits } from '~/helpers/generateFakeVault';
+
+const COLLATERALS = getAllCollateralTypes();
+const liquidationLimits = generateFakeLiquidationLimits();
+
+const common = {
+    components: { VaultLiquidationPanel },
+    data() {
+        return {
+            auctionId: '1',
+            auctionState: 'liquidatable',
+            collateralType: faker.helpers.randomize(Object.values(COLLATERALS)),
+            debtDai: new BigNumber(0),
+            incentiveRelativeDai: new BigNumber(0),
+            incentiveConstantDai: new BigNumber(0),
+            liquidationLimits,
```

Comment:
I also suggest to not hardcode this parameters here (including `network`) but take it from a fake vault